package.json

```diff
@@ -18,7 +18,7 @@
     "speccy:watch": "npx nodemon --watch 'api/swagger.json' --exec 'npm run speccy'",
     "speccy": "npx speccy serve -p 8001 api/swagger.json",
     "start": "npm run prod:server",
-    "test": "NODE_ENV=test npx jest --coverage --verbose",
```

Comment:
it seems you added mocha and chai without documenting it in the pr?

Reply:
You're absolutely right. I should have mentioned it in the PR. I'm new to contributing to online projects outside of work -- I'll get that updated.